test/cookbooks/test/attributes/default.rb

```diff
@@ -0,0 +1 @@
+override['chef_client_updater']['version'] = '13.5.17'
```

Comment:
You'll want to override it with a release from the stable channel. This release might not be around forever and then the tests will start to fail.

Reply:
Updated in amended commit to `13.6.0`